Spectrum/BetaQuest.cs

```diff
@@ -705,81 +364,444 @@ public static short[] GetVAR_Entrances()
+                { 0x05B0, "LW Beyond Mido -> LW Scrubs Grotto" },
+                { 0x05B4, "SFM Entryway -> SFM Wolfos Grotto" },
+                { 0x05B8, "Hyrule Castle Grounds -> HC Storms Grotto" },
+                { 0x05BC, "2 Scrub Grotto" },
+                { 0x05C0, "Hyrule Field -> HF Tektite Grotto" },
+                { 0x05C4, "LW Beyond Mido -> Deku Theater" },
+                { 0x05C8, "Kak Impas Ledge -> Kak Impas House Back" },
+                { 0x05CC, "(Special) Bottom of the Well by Entrance" },
+                { 0x05D0, "Lon Lon Ranch -> LLR Tower" },
+                { 0x05D4, "LLR Tower -> Lon Lon Ranch" },
+                { 0x05DC, "Kak Impas House Back -> Kak Impas Ledge" },
+                { 0x05E0, "Kokiri Forest -> LW Bridge From Forest" },
+                { 0x05E4, "LLR Talons House after Cucco Game -> LLR Talons House" },
+                { 0x05E8, "(Special) KF Deku Sprout Cutscene" },
+                { 0x05EC, "(Special) Spirit Temple Twinrova Room" },
+                { 0x05F0, "(Special) Zeldas Courtyard after Triforce Cutscene" },
+                { 0x05F4, "Prelude of Light Warp -> Temple of Time" },
+                { 0x05F8, "(Special) GF Caught by Gerudos Again" },
+                { 0x05FC, "Death Mountain Summit -> DMT Cow Grotto" },
+                { 0x0600, "Minuet of Forest Warp -> Sacred Forest Meadow" },
+                { 0x0604, "Serenade of Water Warp -> Lake Hylia" },
+                { 0x0608, "Phantom Ganon Boss Room Blue Warp -> SFM Forest Temple Entrance Ledge" },
+                { 0x060C, "Morpha Boss Room Blue Warp -> Lake Hylia" },
+                { 0x0610, "Twinrova Boss Room Blue Warp -> Desert Colossus From Spirit Lobby" },
+            };
+
+            // Entrances overridden by grottos
+            static readonly Dictionary<short, string> Grotto_Table = new()
+            {
+                { 0x003F, "HF Bomb Grotto near Market Entrance" },
+                { 0x036D, "HF Remote Bomb Grotto" },
+                { 0x00EE, "HF Open Grotto" },
+                { 0x0153, "Kak Open Grotto" },
+                { 0x0175, "ZR Open Grotto" },
+                { 0x01A5, "KF Storms Grotto" },
+                { 0x01AD, "LW Bomb Grotto" },
+                { 0x01ED, "DMT Storms Grotto" },
+                { 0x01F5, "DMC Bomb Grotto" },
+                { 0x024E, "ZD Storms Grotto" },
+                { 0x02B2, "GF Storms Grotto" },
+                { 0x02CE, "SFM Storms Grotto" },
+                { 0x02E2, "ZR Bomb Grotto" },
+                { 0x02E6, "HF Across River Bomb Grotto" },
+            };
         }
-        
+
         [SpectrumCommand(
-            Name = "beta_v",
-            Description = "sets version",
-            Cat = SpectrumCommand.Category.Proto,
+            Name = "bqspoiler",
+            Description = "Outputs Beta Quest entrance spoiler log",
+            Cat = SpectrumCommand.Category.Spectrum,
```

Comment:
```suggestion
            Cat = SpectrumCommand.Category.BetaQuest,
```
`SpectrumCommand.Category` is used to group together commands on the help menu. `SpectrumCommand.Category.Spectrum` is reserved for commands of significant importance since they are displayed first; they're either commands that configure the program itself or are important commands that always have applicable usage.